source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1292,6 +1292,16 @@ See the Driver Sessions Spec for the purpose of this value.
 
 .. _drivers update their topology view in response to errors:
 
+
+Connection Pool Creation
+''''''''''''''''''''''''
+
+After a server check is complete, if the server is determined to be
+`data-bearing
+<https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#data-bearing-server-type>`_,
+if the server in question does not already have a connection pool,
+the driver SHOULD create a connection pool for the server.
```

Comment:
Is this a SHOULD b/c not all drivers use connection pools?

I feel like the workflow we are going for is saying:

- If:
    - The driver supports connection pooling AND
    - The server is data-bearing
- Then:
    - By this point, the driver MUST have a connection pool for the server

Reply:
No reason, I can change to your wording if you like.

Reply:
Yeah, a wording change would be good.

Reply:
This still reads awkwardly to me, how about: "For drivers that support connection pools: after a server check is complete, if the server is data-bearing and it does not already have a connection pool, the driver MUST create one for it"

Reply:
Changed to suggested wording.